test/ctia/entity/asset_test.clj

```diff
@@ -22,8 +21,10 @@
 
 (defn additional-tests [asset-id asset-sample]
   (testing "GET /ctia/asset/search"
+   (let [app (helpers/get-current-app)
+         {:keys [get-in-config]} (helpers/get-service-map app :ConfigService)]
     ;; only when ES store
-    (when (= "es" (p/get-in-global-properties [:ctia :store :asset]))
+    (when (= "es" (get-in-config [:ctia :store :asset]))
```

Comment:
Maybe https://github.com/threatgrid/ctia/pull/965#discussion_r477188465

Reply:
indeed there is no other store implemented yet, so we could simply replace all the patterns`(when (= "es" (get-in-config [:ctia :store store-kw])) body)` by  `body`.

Reply:
And if we need to exclude a potential new store later we could use the `when-not` pattern. WDYT?

Reply:
> And if we need to exclude a potential new store later we could use the when-not pattern. WDYT?

sure!